charts/rancher-turtles/templates/rancher-turtles-components.yaml

```diff
@@ -3366,8 +3366,6 @@ rules: []
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  labels:
```

Comment:
This seems to be removed, but turtles follows best practices for integration with RBAC. We should not change aggregation and use aggregated role instead.

Reply:
@Danil-Grigorev this was removed due to the removal of https://github.com/rancher/turtles/blob/main/config/rbac/manager_role_aggregation_patch.yaml in this PR. And the reason for removal is that I was simply following CAPIs structure and there is no `manager_role_aggregation_patch.yaml` under [config/rbac](https://github.com/kubernetes-sigs/cluster-api/tree/main/config/rbac) folder but rather it is located under [config/default](https://github.com/kubernetes-sigs/cluster-api/tree/main/config/default) which is the same as in this PR.

Reply:
@Danil-Grigorev I have reverted it at the end and left as is. We can come back to it if needed in the future